source/troubleshooting/_android.html.erb

```diff
@@ -1,59 +1,54 @@
 <p>Follow these instructions to connect your Android device to GovWifi.</p>
```

Comment:
I think these "follow these instructions..." headers are just noise now the page is collapsible. I'd delete them.

Same for the Instruction header - I don't think it adds anything if there's no other sections.